src/js/components/stock-movement-wizard/outbound/EditPage.jsx

```diff
@@ -83,12 +84,19 @@ const FIELDS = {
         defaultMessage: 'Product name',
         attributes: {
           formatValue: value => (
-            <span className="d-flex">
-              <span className="text-truncate">
-                {value.name || ''}
+            <Tooltip
```

Comment:
use the showValueTooltip and tooltipValue attributes instead _(see comment below)_